Configs/.config/hypr/keybindings.conf

```diff
@@ -136,6 +136,8 @@ bind = $mainMod, mouse_up, workspace, e-1
 # Move/Resize windows with mainMod + LMB/RMB and dragging
 bindm = $mainMod, mouse:272, movewindow
 bindm = $mainMod, mouse:273, resizewindow
+bindm = Super, Z, movewindow
+bindm = Super, X, resizewindow
```

Comment:
move looks fine but resize still requires mouse input

Reply:
Is press main + x and move cursor not working ?? in my laptop this is work not need to mouse input for resize window i try with mouse then also work